time stamp
# latest object color
uint32 current_object 
# 1 = Red 
# 2 = Green 
# 3 = Blue 
# latest tray 
uint32 current_tray 
#1 = Red
#2 = Green
#3 = Blue 
